File: Iniciante/URI 1006.scala
import java.util.Scanner

object Main {
    
    def main(args: Array[String]) {
        val leitor = new Scanner(System.in)
        val a = leitor.nextDouble()
        val b = leitor.nextDouble()
        val c = leitor.nextDouble()
        val media = ((a * 2) + (b * 3) + (c * 5)) / 10
        printf("MEDIA = %.1f\n", media)
    }
    
}

File: Iniciante/URI 1006.kt
import java.util.Scanner;

fun main(args: Array<String>) {
    val leitor = Scanner(System.`in`);
    
    val a: Double = leitor.nextDouble();
    val b: Double = leitor.nextDouble();
    val c: Double = leitor.nextDouble();
    val media: Double = ((a * 2) + (b * 3) + (c * 5)) / 10;

    println(String.format("MEDIA = %.1f", media));
}